opentelemetry-sdk/CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 ### Added
 - Add instrument validation to `InstrumentBuilder` [#884](https://github.com/open-telemetry/opentelemetry-rust/pull/884).
 - Add `TelemetryResourceDetector` [#899](https://github.com/open-telemetry/opentelemetry-rust/pull/899).
+- Added `delta` temporality to metric counter benchmark tests [#1034](https://github.com/open-telemetry/opentelemetry-rust/pull/1034).
```

Comment:
my suggestion is to add changelog only for end user facing changes, and not required for test/benchmark additions.